optimum/intel/neural_compressor/configuration.py

```diff
@@ -90,3 +93,201 @@ def _create_distillation_config(config: Union[Dict, DistillationConfig]):
                 # "loss_weights": criterion.loss_weights,
             }
         return config
+
+
+class INCQuantizationMethod(str, Enum):
+    RTN = "rtn"
+    GPTQ = "gptq"
+    AutoRound = "autoround"
+
+
+@dataclass
+class INCQuantizationConfigBase(QuantizationConfigMixin):
+    """
+    Base configuration class for quantization parameters
+    """
+
+    quant_method = INCQuantizationMethod.RTN
+
+    def __init__(
+        self,
+        bits: int = 4,
```

Comment:
could make sense to have 8 as default
```suggestion
        bits: int = 8,
```